ansible/roles/common/tasks/main.yml

```diff
@@ -27,6 +27,7 @@
   become: yes
 
 - name: run the equivalent of "apt-get update" as a separate step
+  when: ansible_distribution == "Ubuntu" and ansible_lsb.major_release|int >=16
```

Comment:
@kumarashit do we have the clear information on the OS versions? Should we consider that?

Reply:
Right now we support only 16 and 18. we can add the <= check too

Reply:
Adde the check for Ubuntu 16 and 18. The later versions can be slided as we support new versions